src/khoj/processor/org_mode/org_to_entries.py

```diff
@@ -100,12 +100,13 @@ def convert_org_nodes_to_entries(
                 continue
 
             todo_str = f"{parsed_entry.todo} " if parsed_entry.todo else ""
-            # Prepend filename as top heading to entry
-            filename = Path(entry_to_file_map[parsed_entry]).stem
+
+            # Prepend ancestor headings, filename as top heading to entry for context
```

Comment:
It would be either ancestor headings OR filename, right? It looks like `filename` won't be included if it has ancestors, but correct me if I'm wrong. We should make sure to include them.

Reply:
Yeah. filename is included in the ancestor headings